Src/ILGPU.Tests/ExchangeBufferOperations.cs

```diff
@@ -0,0 +1,204 @@
+﻿using ILGPU.Runtime;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace ILGPU.Tests
+{
+    public abstract class ExchangeBufferOperations : TestBase
+    {
+
+        private const int Length = 32;
+
+        protected ExchangeBufferOperations(ITestOutputHelper output, TestContext testContext)
+            : base(output, testContext)
+        { }
+
+        internal static void CopyKernel(Index1 index, ArrayView<long> data)
+        {
+            data[index] -= 5;
```

Comment:
The kernel name suggests that is copies several elements. However, it seems to manipulate the buffer data in-place. Is this intended? Furthermore, I am wondering whether this is really required in this context since such kernels are already tested in the scope of several other tests? Maybe I am missing the point here 😄

Reply:
The point of the said test was to verify that copying to and from the accelerator actually changed the CPUView if the kernel contained an operation of course. I can try to come up with a more suitable name.

Reply:
I see; that makes sense to me 🤓 . Otherwise, we cannot verify that the actual buffer content has been updated by the GPU. Let's keep those kernels 👍